app/src/main/java/ar/com/wolox/android/training/ui/training/login/LoginActivity.java

```diff
@@ -14,6 +19,16 @@ protected int layout() {
 
     @Override
     protected void init() {
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            Window window = this.getWindow();
```

Comment:
The [kotlin standard functions](https://medium.com/@elye.project/mastering-kotlin-standard-functions-run-with-let-also-and-apply-9cd334b0ef84) are very useful for this cases. We could do something like this:
```
Drawable background = this.getResources().getDrawable(R.drawable.gradient_toolbar);
this.getWindow().run {
            addFlags(WindowManager.LayoutParams.FLAG_DRAWS_SYSTEM_BAR_BACKGROUNDS);
            setStatusBarColor(this.getResources().getColor(android.R.color.transparent));
            setNavigationBarColor(this.getResources().getColor(android.R.color.transparent));
            setBackgroundDrawable(background);
}
```
and we will avoid repeating the `window.` call.